src/main/java/kg/jarkyn/cobspecserver/ListenerSocket.java

```diff
@@ -0,0 +1,31 @@
+package kg.jarkyn.cobspecserver;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+public class ListenerSocket implements Listener {
+    private ServerSocket serverSocket;
+    private boolean accepting = true;
```

Comment:
can I ask why?
It always start with true currently, there's no point starting with false
I am sort of "missing" a setter for this field, if I was to implement, do you think that would be sufficient?